src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar.domain
+
+class Cars(private val cars: List<Car>) {
+    init {
+        require(cars.isNotEmpty()) { "자동차 대수는 1대 이상이여야 합니다" }
+    }
+
+    fun move(conditionProvider: ConditionProvider): RacingCarGameSnapShot {
+        cars.forEach {
+            it.move(conditionProvider.nextCondition())
+        }
+
+        return RacingCarGameSnapShot(
+            cars.map { CarSnapShot(it.name, it.currentPosition) }.toList()
+        )
+    }
+
+    fun getWinnersOfGame(): List<Car> {
+        val winner = cars.sorted().first()
```

Comment:
```suggestion
        val winner = cars.minOf { it }
```
Comparable을 구현할거라면 조금 더 적극적으로 사용해봐도 좋을 것 같습니다 😄